docs/docs/rest-api/public/api/v2/types/container-status.raml

```diff
@@ -0,0 +1,56 @@
+#%RAML 1.0 DataType
+type: object
+additionalProperties: false
+properties:
+  name:
+    type: !include name.raml
+    description: name of the container specification (within the pod)
+  status:
+    type: string
+    enum: [ STAGING, STABLE, DEGRADED, TERMINAL ]
```

Comment:
Thinking about the enum values we can/should have here. If both tasks and podInstances are abstracted to a generic `Instance` internally, then it might be nice to have the same states on both (or a generic subset).

Yet we could as well publish distinct events on API level.

I think I like `STABLE` vs `DEGRADED` more than `Running` vs `Running` (_but unhealthy_) as we have for tasks 👍